hibernate-infinispan/src/main/java/org/hibernate/cache/infinispan/InfinispanRegionFactory.java

```diff
@@ -304,7 +304,11 @@ public InfinispanRegionFactory(Properties props) {
 	}
 
```

Comment:
RegionFactory in master has additional methods that take a Map argument. These methods do not exist in 5.1, so they have been removed in in this commit.